app/renderer/utils/parseDate.ts

```diff
@@ -0,0 +1,62 @@
+import { assertNever } from "app/common/utils"
+
+export type dateAccuracy = "min" | "day"
+
+/**
+ *  Function to parse date to a human redable format
+ *
+ * @export
+ * @param {Date} date
+ * @param {dateAccuracy} accuracy
+ * @param {boolean} [ordinal]
+ * @returns
+ */
+export default function parseDate(date: Date, accuracy: dateAccuracy, ordinal?: boolean): string {
+  const monthNames = ["January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"
+  ]
+
+  const min = date.getMinutes()
+  const hour = date.getHours()
+  const day = date.getDay()
```

Comment:
to get the day of the month we need to use [`date.getDate()`](https://www.w3schools.com/js/js_date_methods.asp) instead of `date.getDay()`